### PR TITLE
Render Piskel inside Gamelab via npm

### PR DESCRIPTION
**Pull request by islemaster**
Makes the new [@code-dot-org/piskel](https://www.npmjs.com/package/@code-dot-org/piskel) npm package an apps dependency, copies its contents to the built apps package (via a `piskel-root` utility so we don't depend on an exact location with `node_modules`), and renders it within an iframe on the animation tab.  No other interaction yet.

screenshot from 2016-06-06 14-28-49

This is a new approach to a problem I originally tried to tackle in #8562.

### CODE REVIEW

File: apps/Gruntfile.js [hunk 1]
```diff
@@ -63,6 +64,7 @@ module.exports = function (grunt) {
 
   var ace_suffix = envOptions.dev ? '' : '-min';
   var dotMinIfNotDev = envOptions.dev ? '' : '.min';
+  var piskelRoot = String(child_process.execSync('`npm bin`/piskel-root')).replace(/\s+$/g,'');
```
Comment (islemaster) [inline apps/Gruntfile.js:67]: The newline in the output of this utility caused issues within `grunt-contrib-copy`; I'll simplify this as soon as I fix up the utility.

File: apps/Gruntfile.js [hunk 2]
```diff
@@ -125,6 +127,12 @@ module.exports = function (grunt) {
         },
         {
           expand: true,
+          cwd: piskelRoot,
+          src: '**',
+          dest: 'build/package/js/piskel/'
```
Comment (islemaster) [inline apps/Gruntfile.js:132]: I'll need to replace this copy step with the webpack equivalent once @pcardune's refactor goes in.

Comment (pcardune) [inline apps/Gruntfile.js:132]: the webpack changes don't touch any of the copy logic. so this will just work.

**Comment by islemaster** [comment]
Assigning now that webpack is merged!

File: apps/src/gamelab/AnimationTab/AnimationTab.jsx [hunk 3]
```diff
@@ -72,20 +56,7 @@ const AnimationTab = React.createClass({
...
-          </div>
-          <div style={styles.editorColumn}>
-            <div className="purple-header workspace-header" style={commonStyles.purpleHeader}>
-              <span>Workspace</span>
-            </div>
-            <div style={styles.editorRegion}>
-              TODO: Piskel editor goes here!
-            </div>
-          </div>
+          <iframe id="piskel-frame" style={styles.editorColumn} src="/assets/js/piskel/index.html" />
```
Comment (islemaster) [inline apps/src/gamelab/AnimationTab/AnimationTab.jsx]: Now that I think about it, this path might not work in environments where we fingerprint assets.  I'll revisit this.

File: apps/Gruntfile.js (hunk 2 above)
Comment (pcardune) [inline apps/Gruntfile.js:132]: do the files in here need to be minified? Or are they already minified?

**Comment by pcardune** [comment]
LGTM

File: apps/Gruntfile.js (hunk 2 above)
Comment (islemaster) [inline apps/Gruntfile.js:132]: They are already minified via Piskel's build process.

As-we-type I am working out a nice way to run debug Piskel inside gamelab (it's not as easy as doing a debug build) but that shouldn't block this change.

**Pull request merged**